### DISCUSSION TITLE
How to validate color in Detox?

### DISCUSSION THREAD

**Discussion by baskarkarthik** [wix/Detox#3283] [Q&A]
Hi Detox Team,
If my app has text or link with different colors (ex: Red, green, etc), how to get the attribute of the color in detox?
are there any ways to validate colors in Detox?

**Comment by d4vidi** [comment]
[`getAttributes()`](https://wix.github.io/Detox/docs/api/actions-on-element/#getattributes) seems to be the API that would be best associated with what you are asking (for example, we have `alpha` and `textSize` there), but technically we don't have that right now.

We could quite easily accept that as an [external contribution](https://wix.github.io/Detox/docs/contributing), if you'd like to opt in on that.

For the time being, I'd consider utilizing [screenshot testing](https://wix.github.io/Detox/docs/api/screenshots/).

**Reply by d4vidi** [reply]
Open an equivalent issue, marked for external contribution: #3297
